mlocal/frags/Makefile.stub

```diff
@@ -73,6 +76,18 @@ $(go_OBJ): $(BUILDDIR)/vendors-done $(libruntime) $(BUILDDIR)/config.h
 	@rm -f $(go_OBJ)
 	$(V)export BUILDDIR=`readlink -f $(BUILDDIR)` && cd $(SOURCEDIR)/src/pkg/buildcfg && go generate
 
+# bash_completion file
+$(bash_completion): $(go_OBJ)
+	@rm -f $(bash_completion)
+	@mkdir -p  $(BUILDDIR)/etc/bash_completion.d 
+	$(V)export BUILDDIR=`readlink -f $(BUILDDIR)` && cd $(SOURCEDIR)/src/docs && go generate
```

Comment:
As I said, here we can just call `go run ...` directly, similarly to how we do for `singularity.conf` generation.